### ISSUE TITLE
entering unicode into the "Create new page" field throws errors

### ISSUE THREAD

**Issue by tmaybe** [chimecms/chime#224]
When I enter `૮(꒦ິ ˙̫̮ ꒦ິ)ა` into the *Create new page* field, first I get a

`UnicodeEncodeError: 'ascii' codec can't encode character u'\u0aee' in position 0: ordinal not in range(128)`

and every time I try to load the task's main index page after that, I get a

`GitCommandError: 'git reset --hard' returned exit status 128: fatal: Unable to create '/Users/tma/Documents/repos/chime/repo-f9e062ba-tomas%40codeforamerica.org/.git/index.lock': File exists.`

**Comment by tmaybe** [comment]
The file is created though...
```
ls repo-f9e062ba-tomas%40codeforamerica.org/
_task.yml				૮(꒦ິ ˙̫̮ ꒦ິ)ა.markdown
```

**Comment by tmaybe** [comment]
This is fixed as of #280
